### PR TITLE
PDF Preview

### PR DESCRIPTION
**Pull request by frozenhelium**
We can see the paged preview just by adding `/pdf-preview` to the view url (eg: `/documents/46/v1.0/pdf-preview`)

Note to @sunu: It takes a small amount of time for the pagedjs to convert the preview into the paged view, let me know if there's a way we can add some triggers to let you know that the conversion has been completed.

We can tweak the preview as per the requirement.

CC: @wrynearson

### CODE REVIEW

**Comment by sunu** [comment]
@frozenhelium playwright has auto waiting that might be able to handle the delay. But if that's not the case, we can think of other ways.

File: static/pdf-preview-styles.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  margin-left: 10mm;
+  margin-right: 10mm;
+
+  @bottom-right {
+    content: counter(page);
+  }
+}
+
+h2 {
+  break-before: page;
```
Comment (sunu) [inline static/pdf-preview-styles.css:14]: Page breaks may not be needed before every h2 though. I think we only need forced page breaks after the table of content.

**Review by sunu — APPROVED** [review]
Looks good to me! We can improve styles etc and add a table of content later on.

File: static/pdf-preview-styles.css (hunk 1 above)
Comment (frozenhelium) [reply static/pdf-preview-styles.css:14]: Noted, will improve this one later!

**Review by wrynearson — APPROVED** [review]
Tested locally on /documents/gedi-level-2-b-footprint-canopy-/v1.0/. Looks good so far. We'll look into page breaks and the table of contents soon.

**Pull request merged**